apps/nextjs/src/app/(dashboard)/dashboard/(body)/tournaments/page.tsx

```
@@ -2,10 +2,18 @@ import ComingSoon from "~/components/coming-soon";
 export default function TournamentHistory() {
   return (
-    <ComingSoon title="Tournament History">
-      <div className="flex flex-col items-center">
-        <h2>Tournament History is under construction</h2>
-      </div>
-    </ComingSoon>
+    <div className="m-4 flex flex-col px-4 py-8 sm:px-8 md:px-16 lg:px-80">
+      <ComingSoon title="Tournament History">
+        <div
+          className="flex flex-col items-center"
+          role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)